### DISCUSSION TITLE
ConEmu can be set as the default terminal in Windows. Can I do that with terminus?

### DISCUSSION THREAD

**Discussion by stibinator** [Eugeny/tabby#3781] [General]
Any way of making Terminus the default terminal application for windows, so that if I run cmd or powershell from the windows menu it starts in a terminus window, like the way ConEmu does?
![image](https://user-images.githubusercontent.com/422938/117400478-10d68d80-af46-11eb-9452-9bb66bf0e6e0.png)

**Comment by okuma10** [comment]
Yes I would also like to set tabby as default terminal
